.github/workflows/generate-api-reference.yml

```diff
@@ -0,0 +1,47 @@
+name: Generate API Reference
```

Comment:
question: does this commit directly to master? and if so, is that the best approach?

Thinking it might be nice to have a little bit of extra control in the form of a PR maybe, if that is possible? 

Wdyt?

Reply:
Makes sense, considering some endpoints have incorrect information so we can review them before publishing them.

I'll make changes to it 🙂

Reply:
Nice!